wremnants/theory_tools.py

```diff
@@ -315,7 +315,7 @@ def define_postfsr_vars(df, mode=None):
     if mode is not None:
         # defition of more complex postfsr object 
         # use fiducial gen met, see: https://twiki.cern.ch/twiki/bin/viewauth/CMS/ParticleLevelProducer
-        if mode in ["wlike", "dilepton"]:
+        if "mz" in mode:
```

Comment:
I don't think this is a good idea because we get out of sync with the "mode" as it is used in the datagroups object which is defined without "mz" or "mw"

Reply:
Well I think we have to have options like "wlike_fiducial" "wlike_inclusive" "dilepton_masswindow" "dilepton_inclusive" etc. Putting mz or mw in front makes it a little easier to distinguish without checking all options. But I don't have strong opinions, I'm open to other suggestions